### PR TITLE
Completed Praktikum, Screenshot, and Summary from 08_Javascript-Refreshment

### PR DESCRIPTION
**Pull request by zhafranafif**
- [ ] Completed Praktikum
- [ ] Completed Screenshot
- [ ] Completed Summary

### CODE REVIEW

File: 08_Javascript-Refreshment/praktikum/no1&2.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+var a = 5
```
Comment (syukronarie) [inline 08_Javascript-Refreshment/praktikum/no1&2.js]: kenapa pake var?

**Review by syukronarie — COMMENTED** [review]
please cek komen

File: 08_Javascript-Refreshment/praktikum/no1&2.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+let b = "kampus merdeka"
+const nama = "Budi"
+let terdaftar = false
+let lengkap_arr = [a,b,nama,terdaftar]
```
Comment (syukronarie) [inline 08_Javascript-Refreshment/praktikum/no1&2.js]: variable di js pake camel case convention

File: 08_Javascript-Refreshment/praktikum/no1&2.js (hunk 1 above)
Comment (zhafranafif) [reply 08_Javascript-Refreshment/praktikum/no1&2.js]: kenapa pake var karena ketentuan dari soalnya sendiri mas arie

File: 08_Javascript-Refreshment/praktikum/no1&2.js (hunk 2 above)
Comment (zhafranafif) [reply 08_Javascript-Refreshment/praktikum/no1&2.js]: okee mas arie

File: 08_Javascript-Refreshment/praktikum/no1&2.js (hunk 1 above)
Comment (zhafranafif) [reply 08_Javascript-Refreshment/praktikum/no1&2.js]: untuk variabel var udah saya ganti mas dari var menjadi let

File: 08_Javascript-Refreshment/praktikum/no1&2.js (hunk 2 above)
Comment (zhafranafif) [reply 08_Javascript-Refreshment/praktikum/no1&2.js]: variabel lengkap_arr udah saya ganti menjadi CamelCase convention mas

**Review by syukronarie — APPROVED** [review]
reviewed and approved. tks

**Pull request merged**